Streamlink Twitch GUI/streamlink-twitch-gui.nuspec

```diff
@@ -71,7 +71,7 @@ Example: `choco uninstall streamlink-twitch-gui -y -params '"/Purge"'`
     <dependencies>
       <dependency id="streamlink" version="[1.4.0,)"/>

```

Comment:
Twitch GUI `v1.11.0` requires Streamlink `>=2.0.0`

Reply:
Thanks for catching this! Updated